core/src/test/scala/org/apache/spark/deploy/rest/StandaloneRestSubmitSuite.scala

```diff
@@ -508,7 +508,7 @@ private class DummyMaster(
     exception: Option[Exception] = None)
   extends Actor {
 
-  override def receive = {
+  override def receive: PartialFunction[Any, Unit] = {
```

Comment:
this one is generally OK, right? Not that it's less wrong this way, but just unnecessary.